nautobot/utilities/tables.py

```diff
@@ -61,6 +64,11 @@ def __init__(self, *args, user=None, **kwargs):
         # Init table
         super().__init__(*args, **kwargs)
 
+        if model := getattr(self.Meta, "model", None):
+            # The name nesting is removed when sorting on the UI because it results in rows appearing as children of the wrong parent rows.
+            if issubclass(model, MPTTModel) and self.order_by:
+                self.columns["name"].column.template_code = MPTT_LINK_WITHOUT_NESTING
+
```

Comment:
Don't we still need to add code to *disable* sorting for tables based on `TreeModel`? I thought we'd determined that sorting didn't work for those. Or is that already disabled case-by-case in the individual table definitions?